frontend/lib/src/components/widgets/NumberInput/NumberInput.tsx

```diff
@@ -204,6 +204,17 @@ const NumberInput: React.FC<Props> = ({
       commitValue({ value, source: { fromUi: false } })
     }
 
+    const numberInput = inputRef.current
+    if (numberInput) {
+      // Issue #8867: Disable wheel events on the input to avoid accidental changes
+      // caused by scrolling.
+      numberInput.addEventListener("wheel", e => e.preventDefault())
```

Comment:
Do you know where the wheel feature is implemented? Is it in baseweb or is it a basic html number input feature?

Reply:
Its coming from the basic html number input